src/crimson/osd/backfill_state.cc

```diff
@@ -455,6 +455,11 @@ BackfillState::Done::Done(my_context ctx)
   ls().backfilled();
 }
 
+// -- Crashed
+BackfillState::Crashed::Crashed()
+{
+  ceph_assert("{}: this should not happen" == nullptr);
```

Comment:
nit, might want to use `ceph_abort(...)` in this case.

Reply:
Switched other occurrences as well.